src/GoodsList.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import './App.css';
+
+type Props = {
+  goods: string[];
+};
+
+type State = {
+  copy: string[];
+};
+
+export class GoodsList extends React.Component<Props, State> {
+  state: State = {
+    copy: [...this.props.goods],
```

Comment:
copy не найкраща назва
```suggestion
    visibleGoods: [...this.props.goods],
```